### PR TITLE
Makes tad repairable and adds circuit board to req

### PR DESCRIPTION
**Pull request by Notamaniac**
## About The Pull Request
Allows the Tadpole to be repaired using a circuit, orderable from requisitions for 750 points.

## Why It's Good For The Game
Given that the Tadpole is much more easy to disable nowadays, there's more of a case to allow to be repaired. There is an overall shift to less Tadpole centric gameplay, so ultimately this is up to the maints to deny. Garygoles exist now to let xenoes know if someone is trying to steathily repair the Tadpole, and the circuit board takes time to install, giving xenoes an opportunity to interrupt the repair. Price was sort of eyeballed - so it wouldn't be hideously expensive, but enough to hurt req's bottom line.

## Changelog
:cl:
add: Adds Tadpole circuits to requsitions for 750 points each, which can be used to repair the Tad after it's been disabled
/:cl:

### CODE REVIEW

**Comment by Derrick2Electricboogaloo** [comment]
Exactly this. This is what I have wanted to see for a very long time.

**Comment by SimplySmiley123** [comment]
Fucking. Finally.

**Comment by SimplySmiley123** [comment]
> Please stop deployable shields from being deployed on the tad too, or else the previous nerf means nothing and this is really a straight tad buff.

Those shields are limited.
6 if I remember right, no openable/closable variant so can't put razor on if you wanna leave or gotta pick up and redeploy each and every time. And for Tads where cades did matter, there's too little of these to cover the whole thing so gotta be strategic a little.

They're not that big of a deal and don't invalidate the nerf at all.
If anything gives something to think about where to put them, where they're gonna be needed on the Tad, if the TO knows about them to begin with.

**Comment by iforgotmeword** [comment]
i don't think this would change much about tad...

mean sure it's nice to have don't get me wrong.
But just breaking tad has effectively made tad useless even if repaired
by one killing the TO (most likely perma too because no one gives a dammed about saving him)
and xenos being able to freely melt everything in tad

also Tad just useless when being reused 
it more or less being used to drop supplies before leaving tad on the ship for evac
it's a alamo that is only used one time before being discarded at best

so having it be able to be fixed is a ???
it's a buff but kind of feels like making RO unable to be perma like it's nice but how likely is it to come into effect?

the major problem with TO i see right now is
it's boring as all hell
it's a groundside role that handles like a shipside role
personally i say merge tad with CAS so one role is doing both

or just kill tad (what i still want to see)

File: code/modules/shuttle/mini_dropship.dm [hunk 1]
```diff
@@ -205,6 +205,33 @@
 		return
 	return ..()
 
+/obj/machinery/computer/attackby(obj/item/I, mob/user, params)
+	. = ..()
+	if(.)
+		return
+	if(istype(I,/obj/item/circuitboard/tadpole))
+		var/repair_time = 50
+		if(!(machine_stat & BROKEN))
```
Comment (TiviPlus) [inline code/modules/shuttle/mini_dropship.dm:214]: early return pls

File: code/modules/shuttle/mini_dropship.dm [hunk 2]
```diff
@@ -205,6 +205,33 @@
...
+			to_chat(user,"<span class='warning'>The circuits don't need replacing.</span>")
+			return
```
Comment (TiviPlus) [inline code/modules/shuttle/mini_dropship.dm:216]: span class defines pls

**Review by TiviPlus — CHANGES_REQUESTED** [review]
rest lgtm

**Comment by Lumipharon** [comment]
This seems like its going to just encourage the use of tad in which it was deliberately nerfed, since it makes it impossible to permanently destroy, and one dude can just pop to it later and fix it.

**Pull request merged**